rascaline/src/calculators/lode/spherical_expansion.rs

```diff
@@ -57,6 +57,16 @@ pub struct LodeSphericalExpansionParameters {
     pub potential_exponent: usize,
 }
 
+impl LodeSphericalExpansionParameters {
+    fn get_k_cutoff(&self) -> f64 {
+        if self.k_cutoff.is_some() {
+            return self.k_cutoff.unwrap();
+        } else {
+            return 1.2 * std::f64::consts::PI / self.atomic_gaussian_width;
+        };
```

Comment:
Instead of using unwrap just after checking if it is some, you can use something like this:
```suggestion
        if let Some(k_cutoff) = k_cutoff {
            return k_cutoff;
        } else {
            return 1.2 * std::f64::consts::PI / self.atomic_gaussian_width;
        };
```

or something like

```suggestion
        return self.k_cutoff.unwrap_or(1.2 * std::f64::consts::PI / self.atomic_gaussian_width);
```